README.md

```diff
@@ -172,33 +137,35 @@ you would pandas.
 
-The `modin.pandas` DataFrame is an extremely light-weight parallel DataFrame. Modin 
-transparently distributes the data and computation so that all you need to do is
-continue using the pandas API as you were before installing Modin. Unlike other parallel
-DataFrame systems, Modin is an extremely light-weight, robust DataFrame. Because it is
-so light-weight, Modin provides speed-ups of up to 4x on a laptop with 4 physical cores.
+The `modin.pandas` DataFrame is an extremely light-weight parallel DataFrame. 
+Modin transparently distributes the data and computation so that you can continue using the same pandas API
+while working with more data faster. Because it is so light-weight,
+Modin provides speed-ups of up to 4x on a laptop with 4 physical cores.
 
 In pandas, you are only able to use one core at a time when you are doing computation of
-any kind. With Modin, you are able to use all of the CPU cores on your machine. Even in
-`read_csv`, we see large gains by efficiently distributing the work across your entire
-machine.
+any kind. With Modin, you are able to use all of the CPU cores on your machine. Even with a
+traditionally synchronous task like `read_csv`, we see large speedups by efficiently
+distributing the work across your entire machine.
 
 ```python
 import modin.pandas as pd
 
 df = pd.read_csv("my_dataset.csv")
 ```
 
-#### Modin is a DataFrame designed for datasets from 1MB to 1TB+ 
+#### Modin is a DataFrame designed for datasets from 1MB to 1TB+
+
+Often data scientists have to use different tools
+for operating on datasets of different sizes. Processing large dataframes with pandas
+is slow, and pandas does not support working with dataframes that are too large to fit
+into the available memory. As a result, pandas workflows that work well
+for prototyping on a few MBs of data do not scale to tens or hundreds of GBs (depending on the size
+of your machine). Modin supports operating on data that does not fit in memory, so that you can comfortably
+work with hundreds of GBs without worrying about substantial slowdown or memory errors.
 
-We have focused heavily on bridging the solutions between DataFrames for small data 
-(e.g. pandas) and large data. Often data scientists require different tools for doing
-the same thing on different sizes of data. The DataFrame solutions that exist for 1KB do
-not scale to 1TB+, and the overheads of the solutions for 1TB+ are too costly for 
-datasets in the 1KB range. With Modin, because of its light-weight, robust, and scalable
-nature, you get a fast DataFrame at small and large data. With preliminary [cluster](https://modin.readthedocs.io/en/latest/using_modin.html#using-modin-on-a-cluster)
-and [out of core](https://modin.readthedocs.io/en/latest/out_of_core.html)
-support, Modin is a DataFrame library with great single-node performance and high
+With preliminary [cluster](https://modin.readthedocs.io/en/latest/getting_started/using_modin/using_modin_cluster.html)
```

Comment:
Is this considered preliminary? 

Can this last sentence be merged to a single paragraph with the previous paragraph?